schematic/src/parsecmd.c

```diff
@@ -168,10 +167,6 @@ parse_commandline(int argc, char *argv[])
                     s_post_load_expr);
         break;
 
-      case 'p':
-        auto_place_mode = TRUE;
```

Comment:
The variable `auto_place_mode` is no longer needed. It is present in two more places.